topic-operator/src/test/java/io/strimzi/operator/topic/KafkaImplTest.java

```diff
@@ -175,7 +177,7 @@ public void testTopicMetadataBothNotFound(VertxTestContext testContext) {
                 Either.ofRight(new UnknownTopicOrPartitionException())));
 
         KafkaImpl impl = new KafkaImpl(admin, vertx);
-        impl.topicMetadata(new TopicName("test")).onComplete(testContext.succeeding(topicMetadata -> testContext.verify(() -> {
+        impl.topicMetadata(new Reconciliation("test", KafkaTopic.RESOURCE_KIND, "test", "test"), new TopicName("test")).onComplete(testContext.succeeding(topicMetadata -> testContext.verify(() -> {
```

Comment:
Should we use here and in the other places in this file the DUMMY_RECONCILIATION?